uisdk/src/main/res/drawable/uisdk_gradient_approved.xml

```diff
@@ -2,7 +2,7 @@
     android:shape="rectangle">
     <gradient
         android:angle="0"
-        android:endColor="@color/lightGrey"
-        android:startColor="@color/lightGrey"
+        android:endColor="@color/khTransparentSecondaryColor"
```

Comment:
I suggest using `KhButtonEnabledColor`

Change the name of the file to `uisdk_gradient_disabled.xml `

Reply:
Done